src/axom/inlet/tests/inlet_object.cpp

```diff
@@ -428,6 +426,114 @@ TEST(inlet_object, implicit_conversion_primitives)
   EXPECT_EQ(arr, expected_arr);
 }
 
+TEST(inlet_dict, basic_dicts)
+{
+  std::string testString =
+    "foo = { [\"key1\"] = 4, [\"key3\"] = 6, [\"key2\"] = 10}";
```

Comment:
Would a string literal be better for defining these types of tests?
See: https://en.cppreference.com/w/cpp/language/string_literal

Reply:
Alternatively, you could use single quotes for the internal Lua strings, e.g.
```cpp
     "foo = { ['key1'] = 4, ['key3'] = 6, ['key2'] = 10}"; 
```

Reply:
I think the single quotes are a better option for consistency with Inlet's other tests - I was not aware that Lua allows for both single- and double-quoted strings like Python.